src/core/tests/library/RuntimeCompositor.py

```diff
@@ -41,9 +41,11 @@
 
 
 class RuntimeCompositor(object):
```

Comment:
Why is the runtime compositor needed to unit test Bootstrapper? Two reasons not to do this:
- Runtime compositor is meant for tests requiring most of the classes to be initialized. Bootstrapper does not need that. I'd just have a new class Test_Bootstrapper and have the sudo tests in there with minimal initialization of bootstrapper.
- The way this is done is it unnecessarily overloads RuntimeCompositor for all tests while only catering to the needs of a single class (and not creating anything truly shared for all the tests).

Reply:
not exactly the ask here, the runtime compositor initialize all necessary objects, this is a set up class so it's bootstrapper. all other test files are using compositor for initiation and for unit testings